texturepacker/pluginsrc/com/defold/bob/pipeline/tp/AtlasBuilder.java

```diff
@@ -45,28 +46,19 @@
 
 @ProtoParams(srcClass = AtlasDesc.class, messageClass = AtlasDesc.class)
 @BuilderParams(name="TexturePackerAtlas", inExts=".tpatlas", outExt = ".a.texturesetc")
-public class AtlasBuilder extends Builder<Void> {
+public class AtlasBuilder extends ProtoBuilder<AtlasDesc.Builder> {
 
     static final String TEMPLATE_PATH = "texturepacker/editor/resources/templates/template.tpatlas";
 
     @Override
-    public Task<Void> create(IResource input) throws IOException, CompileExceptionError {
-
-        // Since these template files currently get compiled by bob
-        if (input.getPath().equals(TEMPLATE_PATH))
-        {
-            Task.TaskBuilder<Void> taskBuilder = Task.<Void>newBuilder(this);
-            return taskBuilder.build();
-        }
```

Comment:
we don't build all the files in all the folders anymore, so we don't need this workaround